packages/appsync-modelgen-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -198,6 +235,68 @@ export class AppSyncSwiftVisitor<
           {},
         );
       }
+
+      // mutating functions for updating/deleting
+      Object.entries(obj.fields).forEach(([fieldName, field]) => {
+        const connectionHasOneOrBelongsTo: boolean = field.connectionInfo
+          ? field.connectionInfo.kind === CodeGenConnectionType.HAS_ONE || field.connectionInfo.kind === CodeGenConnectionType.BELONGS_TO
+          : false;
+        if (connectionHasOneOrBelongsTo) {
+          // lazy loading - create setter functions for LazyReference
+          let fieldName = this.getFieldName(field);
+          let capitalizedFieldName = fieldName.charAt(0).toUpperCase() + fieldName.slice(1);
+          structBlock.addClassMethod(
+            `set${capitalizedFieldName}`,
+            null,
+            `self._${fieldName} = LazyReference(${fieldName})`,
+            [
+              {
+                name: `_ ${this.getFieldName(field)}`,
```

Comment:
thsi is intentional for the mutating function: 
```swift
public mutating func setPost(_ post: Post4V2?) {
    self._post = LazyModel(element: post)
}
```
the `_` represents the unnamed parameter to avoid duplication of "post" in the call site. Developers don't have to specific that it is a post that they're passing in:`setPost(post)`. without it, will be `setPost(post: post)`